src/ado/reprun_dataline.ado

```diff
@@ -33,28 +33,20 @@ cap program drop   reprun_dataline
       *Get the states to be checked
       local srng "`c(sortrngstate)'"
       datasignature
-      local dsig "`r(datasignature)'"
+      local dsum "`r(datasignature)'"
```

Comment:
If we always replace this by the checksum below, why do we even keep this?

Reply:
Good point